PlexRequests.UI/Views/Admin/Sonarr.cshtml

```diff
@@ -157,11 +166,44 @@
                     generateNotify("Something went wrong!", "danger");
                 }
             });
+            </text>
         }
-        </text>
-        }
 
 
+
+        @if (!string.IsNullOrEmpty(Model.RootFolder))
+        {
+            <text>
+
+            console.log('Hit root folders..');
+
+            var rootFolderSelected = @Model.RootFolder;
+            if (!rootFolderSelected) {
+                return;
+            }
+            var $form = $("#mainForm");
+            $.ajax({
+                type: $form.prop("method"),
+                data: $form.serialize(),
+                url: "sonarrrootfolders",
+                dataType: "json",
+                success: function(response) {
```

Comment:
There is a bug here where if we keep pressing the Get RootFolders it will keep appending to the list :)